### PR TITLE
Layout variations

### PR DESCRIPTION
**Pull request by katzmo**
Implements https://github.com/moreonion/campaignion/pull/245

### CODE REVIEW

File: template.php [hunk 1]
```diff
@@ -21,6 +21,22 @@
 include 'includes/theme_webform_managed_file.inc';
 include 'includes/theme_webform_time.inc';
 
+/**
+ * Make theme layouts available.
+ */
+function campaignion_foundation_campaignion_layout_info() {
```
Comment (alex-mo) [inline template.php:29]: Isn't there a `Implements hook_campaignion_layout_info().` missing?

**Review by alex-mo — COMMENTED** [review]
Looks OK to me. Disclaimer: I am not at all familiar with layout variations.

File: template.php (hunk 1 above)
Comment (katzmo) [reply template.php:29]: Maybe. Maybe not. I feel like [Drupal docstings](https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards) are science of their own that I have not mastered yet.

Comment (torotil) [reply template.php:29]: Yep, proper style would be to have `Implements …` as the first line in the comment.

Comment (katzmo) [reply template.php:29]: Only for this hook? Or everything else in the file that is not a preprocess or custom function, too?

Comment (torotil) [reply template.php:29]: @katzmo It’s the standard for all [hook implementations](https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#hookimpl).

**Review by torotil — APPROVED** [review]

File: template.php [hunk 2]
```diff
@@ -86,6 +86,14 @@ function campaignion_foundation_process_node(&$vars) {
   $vars['footer_attributes'] = !empty($vars['footer_attributes_array']) ? drupal_attributes($vars['footer_attributes_array']) : '';
 }
 
+/**
+ * Prepares variables for page templates.
+ */
+function campaignion_foundation_process_page(&$vars) {
```
Comment (katzmo) [inline template.php]: Why `process_page` instead of `preprocess_page`?

**Pull request merged**